io.github.voxelcubes.deepqt.yaml

```diff
@@ -0,0 +1,31 @@
+app-id: io.github.voxelcubes.deepqt
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: deepqt
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-config
+  - --filesystem=xdg-cache
+  - --filesystem=home
+  - --share=network
+modules:
+  - pypi-dependencies.yaml
+
+  - name: deepqt
+    buildsystem: simple
+    build-commands:
+      - 'pip3 install --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} deepqt'
```

Comment:
there is no source here.

Reply:
I have completely refactored the manifest now, so this no longer applies. Fixed.